src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/FreeTextSegment.java

```diff
@@ -34,7 +40,7 @@ public static FreeTextSegment fromString(final String edifact) {
 
         final String[] split = Split.byPlus(edifact);
         final String typeCode = split[1];
-        final String freeTextComponent = split[FREE_TEXT_INDEX];
+        final String freeTextComponent = split[4];
```

Comment:
These numbers are values with some semantic content so mostly we're preferring to name the indices.

Reply:
I think we've been reducing these index-related consts and just suppressing the magic number warning.
I think it's clear from this code that index 4 of `split` is the free text (same as index 1 of `split` is the type code), the const to me just confuses things